### PR TITLE
Simple led example

### PR DESCRIPTION
**Pull request by Citrullin**
Add a simple LED on/off example.

### CODE REVIEW

File: examples/LED/LED.ino [hunk 1]
```diff
@@ -1,39 +1,45 @@
...
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+#define LARGE_JSON_BUFFERS 1
+
 #include <Arduino.h>
-#include "Thing.h"
-#include "WebThingAdapter.h"
+#include <Thing.h>
```
Comment (relu91) [inline examples/LED/LED.ino]: logically `Thing.h` is not a system library, why did you change `"` to `<`?

File: examples/LED/LED.ino [hunk 2]
```diff
@@ -46,40 +52,80 @@ void setup(void) {
...
+void do_toggle(const JsonVariant &input) {
+  Serial.println("toggle call");
+  
+  JsonObject inputObj = input.as<JsonObject>();
+  bool state = inputObj["state"];
+
+  Serial.print("state: ");
+  Serial.println(state);
+
+  if (state) {
```
Comment (relu91) [inline examples/LED/LED.ino]: isn't this code redundant? the pin will be written soon after the action handler execution in the loop function.

**Review by relu91 — CHANGES_REQUESTED** [review]
A minor issue below.

File: examples/LED/LED.ino (hunk 1 above)
Comment (Citrullin) [reply examples/LED/LED.ino]: That's true. I don't know. Copy, paste, I think? ^^

File: examples/LED/LED.ino (hunk 2 above)
Comment (Citrullin) [reply examples/LED/LED.ino]: true. I didn't really pay much attention to it.

**Comment by Citrullin** [comment]
@relu91 Updated it.

**Review by relu91 — APPROVED** [review]

**Pull request merged**